supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -219,6 +241,17 @@ Do not use "BIOS" as a generic term to refer to computer firmware. Use "firmware
 
 *See also*: xref:filestore[FileStore], xref:object-store[Object Store]
 
+[discrete]
+[[boolean-dependencies]]
+==== image:images/yes.png[yes] Boolean dependencies (noun)
+*Description*: _Boolean dependencies_ are Boolean expressions (if, and, or, and other expressions) used in the Requires, Conflicts, and Weak dependency directives. Boolean dependencies are also known as Rich dependencies.
```

Comment:
```suggestion
*Description*: _Boolean dependencies_ are Boolean expressions (`if`, `and`, `or`, and other expressions) used in the Requires, Conflicts, and Weak dependency directives. Boolean dependencies are also known as _Rich dependencies_.
```
- I think these are literals and should be in back ticks.
- I'm not sure, but should `Requires`, `Conflicts`, and `Weak` also be in back ticks?
- `Rich dependencies` should also be italicized, I believe.

Reply:
+1
Thanks for catching that.